index.js

```diff
@@ -0,0 +1,7 @@
+const Input = require('./Input');
+const Notes = require('./lib/Notes');
+
```

Comment:
you need to connect and disconnect from mongodb in this file:

```js
const mongoose = require('mongoose');
const Input = require('./lib/Input')
const Note = require('./lib/models/Note');

mongoose.connect('mongodb://localhost:27017/notes', {
  useNewUrlParser: true,
  useUnifiedTopology: true
});

const input = new Input(process.argv);

if(input.valid()) {
  Note.execute(input)
    .then(() => mongoose.connection.close());
} else {
  console.log('Something went wrong');
}
```

Reply:
Corrected missing mongoose in index.js
(Why do we need mongoose in index.js?)